src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -75,6 +81,17 @@ private ResponseEntity postEventInternal(String eventTypeName, String eventsAsSt
         }
     }
 
+    private ResponseEntity processJSONException(final JSONException e, final NativeWebRequest nativeWebRequest) {
+        if (e.getCause() == null) {
```

Comment:
What do you think of having more control over the error message? Instead of using the one provided by the json library we could write our own "Publishing events expect a JSON array".

What scenarios is this `if` clause handling? I mean, is this `if` solving only the `Json array` issue or is it exposing a miriade of cryptic error messages from the json library? I'm afraid this could raise even more questions by users.